### ISSUE TITLE
大佬，你好。麻烦看看下面这个问题，谢谢

### ISSUE THREAD

**Issue by GanHey** [Huangdebo/CAWB#1]
![image](https://user-images.githubusercontent.com/82756842/134609044-f3205d40-d790-4006-8024-e5f85bf20b15.png)

**Comment by GanHey** [comment]
![image](https://user-images.githubusercontent.com/82756842/134609470-8c7d002c-29a8-4f01-93ec-570d31d211b3.png)
好像不支持这个查看学习率

**Comment by Huangdebo** [comment]
（1）那个 batchs=len（data）表示 data 中有多少个 batch，相当于 data 个数 / batch_size
（2）你可以自己在类中添加 get_last_lr 函数，这些都可以自定义的； 其实执行 step()时，返回的就是 last lr 了，last_lr = scheduler.step() 即可

**Comment by GanHey** [comment]
好的  解决了，谢谢大佬！
